osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -81,6 +102,9 @@ private void load(OsuConfigManager config)
             skinDropdown.Bindable = dropdownBindable;
             skinDropdown.Items = skins.GetAllUsableSkins().ToArray();
 
+            currentSkin = skins.CurrentSkin.GetBoundCopy();
```

Comment:
Is there a reason this is not using the `dropdownBindable` instead? (saves for another bindable copy)

Reply:
I prefer having it separated.

Reply:
Looks better now after moving implementation to own class.